buildpipeline/pipeline.json

```diff
@@ -32,12 +33,14 @@
         {
           "Name": "Core-Setup-Linux",
           "Parameters": {
-            "PB_DockerOS": "ubuntu.14.04",
             "REPO_ID": "562fbfe0b2d7d0e0a43780c4",
             "REPO_USER": "dotnet",
             "REPO_SERVER": "azure-apt-cat.cloudapp.net",
             "CLI_NUGET_FEED_URL": "https://dotnet.myget.org/F/cli-deps/api/v2/package",
-            "CLI_NUGET_SYMBOLS_FEED_URL": "https://dotnet.myget.org/F/cli-deps/symbols/api/v2/package"
+            "CLI_NUGET_SYMBOLS_FEED_URL": "https://dotnet.myget.org/F/cli-deps/symbols/api/v2/package",
+            "PB_EnvVars": "-e CLI_NUGET_FEED_URL -e CLI_NUGET_SYMBOLS_FEED_URL ",
+            "PB_DockerOS": "ubuntu.14.04",
```

Comment:
Is there consumer of PB_DockerOS after this change?  Seems like it could be removed.